Investigate failing `tav` test on Jenkins

Investigate failing `tav` tests on Jenkins.

```
                 -- running test "node test/instrumentation/modules/mysql/pool-release-1.js" with mysql
14:50:33 module.js:491
14:50:33     throw err;
14:50:33     ^
14:50:33 
14:50:33 Error: Cannot find module 'bignumber.js'
14:50:33     at Function.Module._resolveFilename (module.js:489:15)
14:50:33     at Function.Module._load (module.js:439:25)
14:50:33     at Function.Module._load (/app/node_modules/require-in-the-middle/index.js:22:24)
```

--- a/test/script/docker/run_docs.sh
+++ b/test/script/docker/run_docs.sh
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-docker build -t node_docs --build-arg NODE_VERSION='latest' .
+docker build --pull -t node_docs --build-arg NODE_VERSION='latest' .
 docker run \
   -e CI=true \
   --rm node_docs \

--- a/test/script/docker/run_tests.sh
+++ b/test/script/docker/run_tests.sh
@@ -20,7 +20,7 @@ else
   CMD='npm test'
 fi
 
-docker-compose build --build-arg NODE_VERSION=$1 node_tests
+docker-compose build --pull --build-arg NODE_VERSION=$1 node_tests
 docker-compose run \
   -e NODE_VERSION=${NODE_VERSION} -e TAV=${TAV_VERSIONS} -e CI=true \
   -v ${npm_cache}:${docker_npm_cache} \

